libcaf_core/caf/flow/step/ignore_elements.hpp

```diff
@@ -0,0 +1,46 @@
+// This file is part of CAF, the C++ Actor Framework. See the file LICENSE in
+// the main distribution directory for license terms and copyright or visit
+// https://github.com/actor-framework/actor-framework/blob/master/LICENSE.
+
+#pragma once
+
+#include "caf/detail/ring_buffer.hpp"
+#include "caf/fwd.hpp"
+
+#include <cstddef>
```

Comment:
Header hygiene: we shouldn't include unnecessary headers.

```suggestion
#include "caf/fwd.hpp"
```